testpackage/tests/restart_read/restart_read.cfg

```diff
@@ -2,17 +2,18 @@ project = Flowthrough
 propagate_field = 0
 propagate_vlasov_acceleration = 0
 propagate_vlasov_translation = 1
-dynamic_timestep = 1
+dynamic_timestep = 0
```

Comment:
Why?

Reply:
The new _acc restart tests handle dynamic timestepping.

Reply:
Do we really need both? Can't we just run one restart test? Also, but this goes beyond this PR, note for further development, we should test restart including e.g. ionosphere.

Reply:
I agree that an ionosphere restart would probably be a good idea. 
Restart tests cost like half a second to run, and it's a lot easier to debug differences in the restart state without having ACC+FS active. At the same time, I think we need a test which verifies that they run in a sane manner after a restart.
I guess, if we wanted, we could do just one write and restart from it with different settings, but that might get confusing.

Reply:
Production-relevant restarting could be tested off the final state of the Ionosphere* test for example. But then maybe again not in this PR? :)

Reply:
Again, it's a question of how it would be good to have tests which allow answering not only the question "does this run?" but also  "*why* does this not run?". If there's a bug, it's a lot easier to debug from a 2D simulation IMO. But I agree that we could add an Ionosphere_restart test, I just think this has value as well.

Reply:
Yep, in the interest of wrapping this PR up I'll resolve this discussion too. It is production-critical that we can restart with all the bells and whistles, and previous experience has shown that we can forget some ionosphere solver variables, or connectivity data, or whatever. But again, not here, resolving this now. Pinpointing individual solvers in this test is useful too.